src/configuration/GPG/GnuPG/gpg.conf

```diff
@@ -205,8 +205,7 @@ keyserver hkp://keys.gnupg.net
 # photo-viewer "metamail -q -d -b -c %T -s 'KeyID 0x%k' -f GnuPG"
 
 
-personal-digest-preferences SHA256
```

Comment:
What's the rationale for changing these parameters ? SHA256 is still considered safe, as far as I know...